internal/container/controller.go

```diff
@@ -203,6 +224,84 @@ func (c Controller) ContainerFiles(id string, filespec string) (files []string)
 	return strings.Split(string(stdout), "\n")
 }
 
+func (c Controller) DownloadFile(id string, src string, dest string) {
+	if id == "" {
+		panic("Must pass in non-empty id")
+	}
+	if src == "" {
+		panic("Must pass in non-empty src")
+	}
+	if dest == "" {
+		panic("Must pass in non-empty dest")
+	}
+
+	cmd := []string{"mkdir", "/var/opt/sql/backup"}
+	c.runCmdInContainer(id, cmd)
+
+	_, file := filepath.Split(src)
+
+	// Wget the .bak file from the http src, and place it in /var/opt/sql/backup
```

Comment:
> .bak file [](http://example.com/codeflow?start=12&length=10)

this package isn't meant to be SQL backup file specific is it?

Reply:
No, the PR is scoped to enable .bak files to get feedback on whether we are on the right track here.

I've added issue:

[https://github.com/microsoft/go-sqlcmd/issues/210](https://github.com/microsoft/go-sqlcmd/issues/210)